doc/man/8/rbd.rst

```diff
@@ -576,7 +576,11 @@ Commands
   details for every mirror-enabled image in the pool or namespace.
 
 :command:`mirror snapshot schedule add` [-p | --pool *pool*] [--namespace *namespace*] [--image *image*] *interval* [*start-time*]
-  Add mirror snapshot schedule.
+  Add mirror snapshot schedule. The ``interval`` can be specified in
+  days, hours, or minutes using the d, h, m suffix respectively.
+  The ``start-time`` is a time string in ISO8601 format. Not providing the
```

Comment:
@zdover23 Just for my education, is the spelling without the space conventional?
```suggestion
  The ``start-time`` is a time string in ISO 8601 format. Not providing the
```